compiler/rustc_middle/src/mir/spanview.rs

```diff
@@ -262,8 +262,8 @@ pub fn terminator_kind_name(term: &Terminator<'_>) -> &'static str {
     match term.kind {
         Goto { .. } => "Goto",
         SwitchInt { .. } => "SwitchInt",
-        Resume => "Resume",
-        Terminate => "Terminate",
+        UnwindResume => "Resume",
+        UnwindTerminate => "Terminate",
```

Comment:
should this be changed too?

Reply:
Ah yeah we should probably keep this consistent.